### PR TITLE
CA-304345: Integrate xsconsole with Travis CI

### PR DESCRIPTION
**Pull request by AaronRobson**

### CODE REVIEW

**Comment by gaborigloi** [comment]
I've closed and re-opened this PR to trigger a Travis build.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+language: python
+
+python:
+  #- 2.5 #Travis was unable to access Python<=2.5
+  - 2.6
```
Comment (gaborigloi) [inline .travis.yml]: I think it should be sufficient to test with 2.7, because that is the version we use in newer XenServers?

Comment (AaronRobson) [reply .travis.yml]: If I recall correctly v2.5 was what I saw on one of our test servers I orchestrated recently. I'll double check when I am next in. We may need to ensure we use the latest versions internally.

Comment (AaronRobson) [reply .travis.yml]: You are correct , I misread my notes 2.7.5 is what we use on our test servers, I'll remove the test for 2.6.

**Comment by AaronRobson** [comment]
Thank you for your approval @gaborigloi, did you also mean to merge or did you want someone else's input beforehand?

**Review by gaborigloi — APPROVED** [review]
Yes, someone else should also review it, if possible, because I do not know much about xsconsole itself.

**Comment by AaronRobson** [comment]
@rosslagerwall and @srowe I wonder if you would review this PR.

**Pull request closed without merging**